Xcode 7 Beta 5 watchOS Unsupported Architecture

I'm currently unable to build the swift-2.0 branch with the following build command:
`sh build.sh build`. 

I'm using Xcode 7b5. When building Realm for the iPhone Simulator I receive multiple errors, e.g.:  
```
In file included from /Users/__/Downloads/realm-cocoa/Realm/ObjectStore/object_schema.cpp:19:
In file included from /Users/__/Downloads/realm-cocoa/Realm/ObjectStore/object_schema.hpp:22:
In file included from /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/string:434:
In file included from /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__config:23:
In file included from /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.0.sdk/usr/include/unistd.h:71:
In file included from /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.0.sdk/usr/include/_types.h:27:
In file included from /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.0.sdk/usr/include/sys/_types.h:32:
/Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.0.sdk/usr/include/sys/cdefs.h:707:2: error: Unsupported architecture
#error Unsupported architecture
```
I've uploaded the entire output [here](http://pastebin.com/yhga5hRy).

Unfortunately I wasn't able to find out which architecture settings (if any) are incorrect.

I am seeing your latter error with a fresh pull:

```   
 export WATCHOS_DEPLOYMENT_TARGET=2.0
    /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -arch armv7k -dynamiclib -isysroot /Applications/Xcode-beta.app/Contents/Developer/Platforms/WatchOS.platform/Developer/SDKs/WatchOS2.0.sdk -L/Users/alan/src/realm-cocoa/build/DerivedData/Realm/Build/Products/Release-watchos -Lcore -F/Users/alan/src/realm-cocoa/build/DerivedData/Realm/Build/Products/Release-watchos -filelist /Users/alan/src/realm-cocoa/build/DerivedData/Realm/Build/Intermediates/Realm.build/Release-watchos/watchOS.build/Objects-normal/armv7k/Realm.LinkFileList -install_name @rpath/Realm.framework/Realm -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -mwatchos-version-min=2.0 -dead_strip -fembed-bitcode-marker -lrealm-watchos -stdlib=libc++ -fobjc-arc -fobjc-link-runtime -fapplication-extension -single_module -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/alan/src/realm-cocoa/build/DerivedData/Realm/Build/Intermediates/Realm.build/Release-watchos/watchOS.build/Objects-normal/armv7k/Realm_dependency_info.dat -o /Users/alan/src/realm-cocoa/build/DerivedData/Realm/Build/Products/Release-watchos/Realm.framework/Realm
ld: in core/librealm-watchos.a(librealm-WatchOS-prelink.a.o), building for watchOS, but linking in object file built for iOS, for architecture armv7k
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **


The following build commands failed:
	Ld build/DerivedData/Realm/Build/Products/Release-watchos/Realm.framework/Realm normal armv7k
(1 failure)
Nomenclature:realm-cocoa alan$ git status
On branch swift-2.0
Your branch is up-to-date with 'origin/swift-2.0'.
nothing to commit, working directory clean
```

--- a/build.sh
+++ b/build.sh
@@ -14,7 +14,7 @@ set -o pipefail
 set -e
 
 # You can override the version of the core library
-: ${REALM_CORE_VERSION:=0.92.1} # set to "current" to always use the current build
+: ${REALM_CORE_VERSION:=0.92.1.1} # set to "current" to always use the current build
 
 # You can override the xcmode used
 : ${XCMODE:=xcodebuild} # must be one of: xcodebuild (default), xcpretty, xctool

